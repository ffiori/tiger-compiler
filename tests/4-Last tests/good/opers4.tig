let
  var k:int :=0
in
	for i:=0 to 4 do
	for j:=i to 4 do
	printInt(j);
	0
end
