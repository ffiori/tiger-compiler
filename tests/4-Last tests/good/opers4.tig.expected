INT: 0
INT: 1
INT: 2
INT: 3
INT: 4
INT: 1
INT: 2
INT: 3
INT: 4
INT: 2
INT: 3
INT: 4
INT: 3
INT: 4
INT: 4
